mobile/lib/modules/home/views/home_page.dart

```diff
@@ -23,18 +28,27 @@ class HomePage extends HookConsumerWidget {
     final appSettingService = ref.watch(appSettingsServiceProvider);
     var renderList = ref.watch(renderListProvider);
     final multiselectEnabled = ref.watch(multiselectProvider.notifier);
+    final selectionEnabledHook = useState(false);
+
     final selection = useState(<AssetResponseDto>{});
+    final albums = ref.watch(albumProvider);
+    final albumService = ref.watch(albumServiceProvider);
 
     useEffect(
       () {
         ref.read(websocketProvider.notifier).connect();
         ref.read(assetProvider.notifier).getAllAsset();
+        ref.read(albumProvider.notifier).getAllAlbums();
         ref.watch(serverInfoProvider.notifier).getServerVersion();
         return null;
       },
       [],
     );
 
+    selectionEnabledHook.addListener(() {
```

Comment:
Should you need to evict the listener to avoid a memory leak? I think you can evict something by using the `return => () {}` from the `useEffect` hook.

Reply:
Another thought, why don't we change `multiselectEnabled.state` directly?

Reply:
I think this should not be a problem because the hooks are local to the component and should be automatically deleted by gc. But yes, better safe than sorry 😃.

`multiselectEnabled.state` did not work properly. For some reason it does not call `didUpdateWidget` in ImmichAssetGrid (think that's a bug with flutter). Using the hook was an easy workaround.